File: Temporary/template.c
#include <stdio.h>
#include <stdlib.h>

//functions
int add(int a, int b){
	return a+b;
}

int sub(int a, int b){
	return a-b;
}

int main(int argc, char* argv[]){

	//FUNCTION POINTERS
	/*
	int (*ptr)(int, int);
	ptr = &add;

	int ans = ptr(2, 3);

	ptr = &sub;

	int ans2 = ptr(2, 3);

	printf("%d\n", ans2);
	*/


	// printf("%s\n", argv[0]); prints template.exe


	//FILE READ WRITE
	
	// FILE *file = fopen("input.txt", "r");

	// if(file==0){
	// 	printf("could not open\n");
	// }
	// else{
	// 	char x;
	// 	while( (x=fgetc(file)) != EOF ) {
	// 		printf("%c\n", x);
	// 	}
	// 	fclose(file);
	// }
	

	


}

File: Temporary/template.py
filename = "input.txt"

# filename = input()
t = open(filename, 'rt')

lines = t.readlines()

count = 0
ans = 0

for line in lines:
	words = line.split(' ')
	print(words[-1])
	# if curr > 5000:
	# 	count += 1
	# 	ans += curr
	

myfile = open( "bytes_"+filename, 'w+')

myfile.write("%d\n" %count)
myfile.write("%d" %ans)